doc/user-guide/src/installation/other.md

```diff
@@ -38,6 +38,21 @@ $ curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- --profile
 > If you have encountered any problems installing `rustup` with a package manager,
 > please contact the package maintainer(s) for further information.
 
+In general, installing rustup with your package manager means installing a rustup binary to 
+somewhere accessible via your `$PATH`.
+To complete installing Rust, you will need to find out the name of that binary:
+
+- If the binary is named `rustup`, picking a default toolchain would usually be enough.
+  A popular choice is `stable`:
+
+  ```sh
+  rustup default stable
+  ```
+
+- If the binary is named `rustup-init`, simply execute `rustup-init` and follow the instructions.
+  When the installation is complete, make sure that `$HOME/.cargo/bin` is in your `$PATH`,
+  and you should be able to use `rustup` normally.
```

Comment:
@djc Various package managers take slightly different approaches towards installing the rustup package. Some work like [`dnf`](https://developer.fedoraproject.org/tech/languages/rust/further-reading.html) where you're simply getting `rustup-init` and you'll have to run it manually to get `rustup`. Others work like `apt` or [`pacman`](https://wiki.archlinux.org/title/Rust#Arch_Linux_package) where you get a `rustup` binary directly but you still have to pick a default toolchain.

`brew` has just moved from the first to the second, but I think it's still helpful for the user to at least have some clue about what they should probably do after running `foo install rustup`.

Reply:
Okay, I think something like this context should go in the docs rather than the more mechanical/opaque description you had before. Thanks for the explanation!

Reply:
> Okay, I think something like this context should go in the docs rather than the more mechanical/opaque description you had before. Thanks for the explanation!

@djc I agree, however the general description is deliberately written without having to explicitly mention specific package managers. I'll try my best to adjust the wording while maintaining that.

Reply:
@djc Done! How do you like my new wording?